src/core/helpers/package.json

```diff
@@ -33,6 +33,7 @@
     "@guardian/src-foundations": "^3.5.0"
   },
   "devDependencies": {
+    "mini-svg-data-uri": "^1.2.3",
```

Comment:
Does this only get used at build time? If not, do you think it should be a `dependency`?

Reply:
Ideally, the SVG optimisation would happen at build time. There are only ever 5 distinct `background-image` values, so this does not have to be evaluated at runtime.

Reply:
Sorry, I'll rephrase that. 

`mini-svg-data-uri` is used at runtime and therefore I think it should be a `dependency`, rather than a `devDependency`. Is this accurate?